src/Microsoft.AspNet.Identity/IdentityErrorDescriber.cs

```diff
@@ -15,6 +15,15 @@ public virtual IdentityError DefaultError()
                 Description = Resources.DefaultError
             };
         }
+        public virtual IdentityError ConcurrencyFailure()
+        {
+            return new IdentityError
+            {
+                Code = nameof(ConcurrencyFailure),
```

Comment:
Not closely related to this checkin, but would CallerMemberNameAttribute in a parameter constructor of IdentityError be nicer way to achieve this?